system/ee/EllisLab/ExpressionEngine/Library/Filesystem/Filesystem.php

```diff
@@ -319,7 +319,7 @@ public function rename($source, $dest)
 			throw new FilesystemException("Cannot rename, destination already exists: {$dest}");
 		}
 
-		rename(
+		@rename(
```

Comment:
I've dug into the possible effects of silencing this rename and it looks like even if it fails, PHP just returns `false` and only throws a `Warning` if the item trying to be renamed is a directory and that directory already exists.

I'm ok with silencing it if you can add this comment above it so it's clear why it's been silenced:
> // Suppressing potential warning when renaming a directory to one that already exists.

Reply:
np - done